### DISCUSSION TITLE
Application Backup frequency

### DISCUSSION THREAD

**Discussion by milindpatel63** [Kovah/LinkAce#363] [General]
How often does linkace backup runs. It's not mentioned in the docs.

Does it even run periodically or do we need to manage it ourselves?

**Comment by Kovah** [comment]
Hi,
Backups of LinkAce itself are run every day, if they are enabled.
Backups are kept until a total size of 500MB is reached (which can be configured). Otherwise
- 16 daily backups are kept,
- 8 weekly backups are kept,
- 4 monthly backups are kept and
- 2 yearly backups are kept.

More details: https://spatie.be/docs/laravel-backup/v6/cleaning-up-old-backups/overview

**Reply by milindpatel63** [reply]
Ohhkk..i see...
So the cron job which is running every minute is responsible for all of this...
I don't need to setup any other corn job specifically for backups right?

**Reply by Kovah** [reply]
No, only one cron is needed. It handles all scheduled tasks internally.
